fetch genome, map extracted reads and add IGV (#62)
genomic-medicine-sweden/metaval: docs/output.md

- [BLAST](#BLAST) - Run BLASTN or BLASTX
- [Bowtie2](#Mapping) - Map raw Illumina reads to a pathogen genome database or map Illumina reads of specific taxIDs to genomes with positive BLAST hits.
- [minimap2](#Mapping) - Map raw Nanopore reads to a pathogen genome database or map Nanopore reads of specific taxIDs to genomes with positive BLAST hits.

> - [IGV](#IGV) - Report for visualizing reads mapped to the genome

Reviewer: ```suggestion
- [IGV](#IGV) - Report for visualizing reads mapped to the genomes identified.
```

Map Illumina short reads to genomes using `bowtie2` and map Nanopore long reads to genomes using `minimap2`

#### Verify identified species

> Map reads to genomes with BLAST hits or genomes of identified species if skipping BLAST steps (`params.skip_blastn` and `params.skip_blastx`).

Reviewer: ```suggestion
Map reads to genomes based on BLAST hits or to genomes of identified species if the BLAST steps are skipped.(`params.skip_blastn` and `params.skip_blastx`).
```

<details markdown="1">
<summary>Output files</summary>

- `mapping/`
  - `bowtie2/`
    - `align/`

>       - `<sample_id>_<taxid>_aligned_genome_sorted.bam`: BAM file containing short reads that aligned against the genomes with BLAST hits.

Reviewer: ```suggestion
      - `<sample_id>_<taxid>_aligned_genome_sorted.bam`: BAM file containing short reads that were aligned against the genomes based on BLAST hits.
```

      - `<sample_id>_<taxid>_aligned_genome_sorted.bam.bai`: Index of the bam file.
    - `build/`
      - `bowtie2/*.bt2l`: Bowtie2 indices of genomes with BLAST hits
  - `minimap2/`
    - `align/`

>       - `<sample_id>_aligned_genome_sorted.bam`: BAM file containing long reads that aligned against the user-supplied pathogens genomes

Reviewer: ```suggestion
      - `<sample_id>_aligned_genome_sorted.bam`: BAM file containing long reads that were aligned against the user-supplied pathogens genomes
```

      - `<sample_id>_aligned_genome_sorted.bam.bai`: Index of the bam file.
    - `index/`

>       - `*.mmi`: Minimap2 indicies of reference pathogens genomes

Reviewer: ```suggestion
      - `*.mmi`: Minimap2 indices of the reference pathogens' genomes
```

The `pathogens` directory will only be present if `--perform_screen_pathogens` is supplied.

### IGV

> Generate an IGV report to visualize the variants and coverage of mapped reads across genomes.

Reviewer: Is mentioning of variants necessary here since we only look at the coverage?

Author: We did look at variants, for instance HIV and BK virus we saw before. I could remove it if you think it's miss leading.

Reviewer: Let's leave it as it is then.

#### Verify identified species

> IGV visualization of reads assigned to specific species by classifiers.

Reviewer: ```suggestion
IGV visualization of reads that classifiers assigned to specific species.
```

<details markdown="1">
<summary>Output files</summary>

- `IGV/`

>   - `<sample_id>_<taxID>_report.html`: IGV report shows variants and coverage of reads mapped to the genomes.

Reviewer: Same here with the variants.

#### Pathogen screening

> IGV visualization of reads mapped to pathogen genome databases.

Reviewer: ```suggestion
IGV visualization of reads mapped to pathogen genome database.
```
